### ISSUE TITLE
Add filters to transfermarket

### ISSUE THREAD

**Issue by Quotic** [fabfischer/kickbase-plus#34]
Hey Fabian,

I know you are on vacation, enjoy the time! I just collect a few ideas here:

Add filters to transfermarket
- show only players where market value is growing
- show only players without other bids
- show only players with own bid
- show only players without injuries
- show only players who are sold before the next matchday
- show only players who are sold after the next matchday
- **show only players who are sold before the next market value update (22:00)**
- show only players who are sold after the next market value update (22:00)

Sort players on transfermarket:
- Market value
- Growth

Just some ideas, maybe you can evaluate if you find this also useful!

BR

Bastian

**Comment by fabfischer** [comment]
Hi @Quotic ,

I added some new filters. 

Your requested filters
```
show only players who are sold before the next matchday
show only players who are sold after the next matchday
show only players who are sold before the next market value update (22:00)
show only players who are sold after the next market value update (22:00)
```
and `sorting` will need some more time.

But you can check the new filters already. Just visit https://kickbaseplus.fabian-fischer.com/#/transfermarket and let me know, what do you think.

Fabian

**Comment by Quotic** [comment]
@fabfischer thanks for your work, this filters are very helpful. I will continue to share my ideas with you ;P

**Comment by fabfischer** [comment]
Hi @Quotic,

I pushed an update, which includes the filter and sorting. You can check it here: https://kickbaseplus.fabian-fischer.com

Please let me know what you think.

Fabian

**Comment by Quotic** [comment]
Its perfect but in mobile version filter descriptions are too long. Maybe shorten description and remove "show only players who are sold "

**Comment by fabfischer** [comment]
Hi @Quotic ,
all filters should now be fully visible for mobile devices. . You can test it here: https://kickbaseplus.fabian-fischer.com

**Comment by fabfischer** [comment]
Hi @Quotic,

I released it with version 1.6 (#58), so I'm closing this issue.

you can test it here: https://kickbaseplus.fabian-fischer.com/

best,
Fabian
